inference-engine/tests/functional/plugin/shared/src/single_layer_tests/fake_quantize.cpp

```diff
@@ -26,8 +26,8 @@
 /**
  * redefine this seed to reproduce issue with given seed that can be read from gtest logs
  */
-#define BASE_SEED   USE_CLOCK_TIME
-#define NGRAPH_SEED USE_CLOCK_TIME
+#define BASE_SEED   123
+#define NGRAPH_SEED 123
```

Comment:
Just for education, why did you change `USE_CLOCK_TIME` to `123`?

Reply:
Random seed leads to sporadic fails in the test.